chrome/content/zotero/xpcom/file.js

```diff
@@ -1024,4 +1024,8 @@ Zotero.File = new function(){
 		
 		throw e;
 	}
+
+	this.isDropboxDirectory = function(path) {
+		return path.toLowerCase().includes('dropbox');
```

Comment:
`String.prototype.includes()` is Firefox 40, but the last Firefox ESR is 38 and the next (45) isn't [until March](https://www.mozilla.org/en-US/firefox/organizations/faq/), so unfortunately we're stuck with `indexOf()` for now.